boards/px4/fmu-v5/src/stm32_userspace.c

```diff
@@ -0,0 +1,147 @@
+/****************************************************************************
```

Comment:
All of this  could be moved to be common for all STM32 targets; however, would like to keep it here until there is another target that uses it